VRCFaceTracking/Params/Expressions/UnifiedExpressions.cs

```diff
@@ -0,0 +1,235 @@
+﻿namespace VRCFaceTracking.Params
+{
+    // TODO: Create expressions map to relate legacy shapes to new ones
+
+    /// <summary> 
+    /// Represents the type of Shape data being sent by UnifiedExpressionData, in the form of enumerated shapes.
+    /// </summary>
+    /// <remarks>These shapes have a strong basis on the underlying muscular foundations of the entire head including the face, eyes, tongue, and inner mouth expressions.</remarks>
+    public enum UnifiedExpressions
```

Comment:
There are several UnifiedExpressions shapes that go completely unused by any interface, as follows:
```
NasalDilationRight, // Right side nose's canal dilates. Basis on the alar nasalis muscle.
NasalDilationLeft, // Left side nose's canal dilates. Basis on the alar nasalis muscle.
NasalConstrictRight, // Right side nose's canal constricts. Basis on the transverse nasalis muscle.
NasalConstrictLeft, // Left side nose's canal constricts. Basis on the transverse nasalis muscle.

JawBackward, // Pulls the jawbone backwards slightly. Based on the retraction of the temporalis muscle.
JawClench, // Specific jaw muscles that forces the jaw closed. Causes the masseter muscle (visible close to the back of the jawline) to visibly flex.
JawMandibleRaise, // Raises mandible (jawbone).

TongueBendDown, // Pushes tip of the tongue below the rest of the tongue in an arch.
TongueCurlUp, // Pushes tip of the tongue above the rest of the tongue in an arch.
TongueSquish, // Tongue becomes thinner width-wise and slightly thicker height-wise.
TongueFlat, // Tongue becomes thicker width-wise and slightly thinner height-wise.

TongueTwistRight, // Tongue tip rotates clockwise from POV with the rest of the tongue following gradually.
TongueTwistLeft, // Tongue tip rotates counter-clockwise from POV with the rest of the tongue following gradually.

SoftPalateClose, // Visibly lowers the back of the throat (soft palate) inside the mouth to close off the throat.
ThroatSwallow, // Visibly causes the Adam's apple to pull upward into the throat as if swallowing.

NeckFlexRight, // Flexes the Right side of the neck and face (causes the right corner of the face to stretch towards.)
NeckFlexLeft, // Flexes the left side of the neck and face (causes the left corner of the face to stretch towards.)

```

However as far as using them on avatars I left that as a possibility if avatar creators wanted to have the shapes on their avatars for potential future interfaces. What do you all think about having these documented and, if exposed in documentation, how relevant should they be? They are heavily based on facial anatomies and other standards implementing them (such as FACS and additions to it). 

Or really any change to them I'm pretty open about it 😅